cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help

```diff
@@ -43,6 +43,9 @@ OPTIONS
        --debug
            Run in remote debugging mode.
 
+       --with-choreo
+           Package Choreo observability extension in the executable jars created while building.
+
 CONFIG PROPERTIES
        (--key=value)...
            Set Ballerina environment parameters as key/value pairs.
```

Comment:
```suggestion
           Set the Ballerina environment parameters as key/value pairs.
```

Reply:
Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5